Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController.swift

```diff
@@ -223,3 +239,45 @@ extension ConversationInputBarViewController: InformalTextViewDelegate {
         updateNewButtonTitleLabel()
     }
 }
+
+// MARK: - ZMConversationObserver
```

Comment:
```suggestion
// MARK: - ZMConversationObserver

```

Let's put some white space around the MARK annotation.